src/main/resources/application-folio.yml

```diff
@@ -4,6 +4,7 @@ mod-linked-data:
   kafka:
     topic:
       search-bibframe-index: search.bibframe
+      search-bibframe-authorities-index: search.bibframe-authorities
```

Comment:
it should be singular: search-bibframe-authority-index: search.bibframe-authority

Reply:
include it in [MSEARCH-786] (https://folio-org.atlassian.net/browse/MSEARCH-786)
because now it's a part of contract in M-SEARCH